code/game/machinery/computer/communications.dm

```diff
@@ -354,18 +360,19 @@ var/list/shuttle_log = list()
 			update_icon()
 
 		if("SetPortRestriction")
-			if(issilicon(usr))
+
+			if(issilicon(usr) && !is_malf_owner(usr))
 				return
 			var/mob/M = usr
 			var/obj/item/weapon/card/id/I = M.get_id_card()
-			if (I || isAdminGhost(usr))
-				if(isAdminGhost(usr) || (access_hos in I.access) || ((access_heads in I.access) && security_level >= SEC_LEVEL_RED))
+			if (I || isAdminGhost(usr) || issilicon(usr))
+				if(isAdminGhost(usr) || issilicon(usr) || (access_hos in I.access) || ((access_heads in I.access) && security_level >= SEC_LEVEL_RED))
```

Comment:
undocumented change, and an extremely unhealthy one

Reply:
Its not undocumented. The silicon has to be a malf that hacked the machine in order to use it, see line 364.

Reply:
Undocumented change means you've snuck in a new feature without putting it in the changelog. In this case you did not CL that the Malf AI can now lock down the trading port.

Reply:
Updated the changelog to be more descriptive

Reply:
It nor the document mentions this